.gitignore

```diff
@@ -101,11 +101,5 @@ Thumbs.db
 ##########
 .sass-cache
 
-# Things in the core directory that Drupal 8 commits in the repository.
-!web/core/**/*.gz
-
-# Ignore files generated by PhpStorm
-/.idea/
```

Comment:
@froboy why do we remove php storm?

Reply:
also wondering if we should remove the `.vscode` directory

Reply:
Not removing altogether, just removing a dupe.
https://github.com/kanopi/starter-8/blob/15e076fe21821144e6d530042d6eea9b3da6f15c/.gitignore#L67

Reply:
Ahh ok!